common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -140,6 +148,63 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         }
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        final String methodTag = TAG + ":onResume";
+        final Bundle extras = getExtras();
+        Logger.info(methodTag, "extras: " + extras.keySet());
```

Comment:
>Logger.info(methodTag, "extras: " + extras.keySet()); [](http://example.com/codeflow?start=8&length=53)

Hope this is not logging anything secure or private.

Reply:
remove